site/src/pages/HealthPage/WorkspaceProxyPage.tsx

```diff
@@ -0,0 +1,146 @@
+import { useOutletContext } from "react-router-dom";
+import {
+  BooleanPill,
+  Header,
+  HeaderTitle,
+  HealthyDot,
+  Main,
+  Pill,
+} from "./Content";
+import { HealthcheckReport } from "api/typesGenerated";
+import { useTheme } from "@mui/material/styles";
+import { createDayString } from "utils/createDayString";
+import PublicOutlined from "@mui/icons-material/PublicOutlined";
+import Tooltip from "@mui/material/Tooltip";
+import TagOutlined from "@mui/icons-material/TagOutlined";
+import { Alert } from "components/Alert/Alert";
+import { Helmet } from "react-helmet-async";
+import { pageTitle } from "utils/page";
+
+export const WorkspaceProxyPage = () => {
+  const healthStatus = useOutletContext<HealthcheckReport>();
+  const { workspace_proxy } = healthStatus;
+  const { regions } = workspace_proxy.workspace_proxies;
+  const theme = useTheme();
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Workspace Proxy - Health")}</title>
+      </Helmet>
+
+      <Header>
+        <HeaderTitle>
+          <HealthyDot severity={workspace_proxy.severity} />
+          Workspace Proxy
+        </HeaderTitle>
+      </Header>
+
+      <Main>
+        {workspace_proxy.warnings.map((warning) => {
+          return (
+            <Alert key={warning.code} severity="warning">
+              {warning.message}
+            </Alert>
+          );
+        })}
+
+        {regions.map((region) => {
+          const warnings = region.status?.report?.warnings ?? [];
+
+          return (
+            <div
+              key={region.id}
+              css={{
+                borderRadius: 8,
+                border: `1px solid ${theme.palette.divider}`,
+                fontSize: 14,
+              }}
+            >
+              <header
+                css={{
+                  padding: 24,
+                  display: "flex",
+                  alignItems: "center",
+                  justifyContent: "space-between",
+                  gap: 24,
+                }}
+              >
+                <div css={{ display: "flex", alignItems: "center", gap: 24 }}>
+                  <div
+                    css={{
+                      width: 36,
+                      height: 36,
+                      display: "flex",
+                      alignItems: "center",
+                      justifyContent: "center",
+                    }}
+                  >
+                    <img
+                      src={region.icon_url}
+                      css={{ objectFit: "fill", width: "100%", height: "100%" }}
+                      alt=""
```

Comment:
I'm guessing that this is to make it so that the image doesn't display alt text if the image request fails?

Reply:
it is because we use many decorative images https://www.w3.org/WAI/tutorials/images/decorative/.

When decorative images are displayed together with the info, I don't see value in adding an alt to them.

Reply:
Oh, in that case, could you add the `presentation` role to them? I know the behavior isn't supported everywhere, but I feel like it'd make the intent a little more clear

Reply:
I'm ok with that, I just think alt="" for presentation is just good enough